NEWS

```diff
@@ -11,9 +11,156 @@
 ### Features:
 ### Bugfixes:
 
-## 1.18.0-rc2 (December 10, 2024)
-### Features: TBD
-### Bugfixes: TBD
+## 1.18.0-rc3 (December 18, 2024)
```

Comment:
Usually we have a separate list of features and bugs for each RC.

Reply:
since previous rc were mentioning TBD and will not be kept, I am assuming this is ok to proceed like this.